mava/types.py

```diff
@@ -131,7 +131,7 @@ class ObservationGlobalState(NamedTuple):
     agents_view: chex.Array  # (num_agents, num_obs_features)
     action_mask: chex.Array  # (num_agents, num_actions)
     global_state: chex.Array  # (num_agents, num_agents * num_obs_features)
-    step_count: chex.Array  # (num_agents, )
+    step_count: Optional[chex.Array] = None  # (num_agents, )
```

Comment:
Why do we need to set a default here and mark it as optional?

Reply:
It wasn't used in the rest of the code and adding (initially) was a bit unclean so I just made it optional, should i just add it back?